modules/installation-azure-regions.adoc

```diff
@@ -53,9 +53,11 @@ The installation program dynamically generates the list of available Microsoft A
 
 Support for the following Microsoft Azure Government (MAG) regions was added in {product-title} version 4.6:
 
-//* usdodcentral (US DoD Central)
-//* usdodeast (US DoD East)
-* usgovarizona (US Gov Arizona)
-* usgoviowa (US Gov Iowa)
 * usgovtexas (US Gov Texas)
 * usgovvirginia (US Gov Virginia)
+//* usdodcentral (US DoD Central)
+//* usdodeast (US DoD East)
+//* usgovarizona (US Gov Arizona)
+//* usgoviowa (US Gov Iowa)
+
+You can reference all available MAG regions in the link:https://azure.microsoft.com/en-us/global-infrastructure/geographies/#geographies[Azure documentation]. Other provided MAG regions are expected to work with {product-title}, but have not been tested.
```

Comment:
LGTM. I was only able to confirm usgovtexas and usgovvirginia with our current access, although all regions are expected to work.